yarn.lock

```diff
@@ -1935,7 +1935,7 @@
   dependencies:
     "@babel/runtime" "^7.19.0"
 
-"@mui/lab@^5.0.0-alpha.115":
+"@mui/lab@5.0.0-alpha.115":
```

Comment:
use of the alpha version ?

Reply:
It is a bit confusing. I'm only using LoadingButton component referenced in mui-core documentation (https://mui.com/material-ui/react-button/#loading-button)), with no details about using an alpha version on a third party lib (no stable one). Nonetheless i should have seen it earlier. I will investigate later